.github/workflows/toxicity-check.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Toxicity check'
+on: 
+  issue_comment:
+    types: ['created', 'edited', 'deleted']
+  pull_request_review:
+  pull_request_review_comment:
+  issues:
+    types: ['opened', 'edited', 'deleted', 'closed', 'reopened', 'locked', 'unlocked']
+  pull_request:
+    types: ['opened', 'edited', 'closed', 'reopened', 'locked', 'unlocked']
+
+jobs:
+  hello_world_job:
```

Comment:
is `hello_world_job` really the best name for the job? :smile:

Reply:
Hehe, that is a left-over from the tutorial I based the bot off of. 😅